### PR TITLE
Make additional/custom resources available

### PR DESCRIPTION
**Pull request by mulhod**
Addresses #88 and #89 and supersedes PR #91.

I still want to do some testing with it, so this is not ready for review.

### CODE REVIEW

File: gridmap/job.py [hunk 1]
```diff
@@ -238,6 +248,12 @@ def native_specification(self):
             ret += " -q {}".format(self.queue)
         if self.gpu:
             ret += " -l gpu={}".format(self.gpu)
+        if self.h_vmem:
+            ret += " -l h_vmem={}".format(self.h_vmem)
+        if self.h_rt:
+            ret += " -l h_rt={}".format(self.h_rt)
+        if self.resources:
+            ret += " " + " ".join([" -l {}".format(x) for x in self.resources])
```
Comment (desilinguist) [inline gridmap/job.py:256]: Is there a reason why this shouldn't just be?
```suggestion
            ret += "".join([" -l {}".format(x) for x in self.resources])
```

Comment (mulhod) [reply gridmap/job.py:256]: No, I believe you're correct. Let me commit the suggestion.

**Comment by mulhod** [comment]
While this is finishing up CI again, I'd like to mention that I have run some non-trivial tasks using this branch today (SKLL unit tests). I'm fairly confident that it works as intended.

**Comment by desilinguist** [comment]
I guess the documentation will update automatically when we merge since all we have is API documentation?

**Comment by mulhod** [comment]
Ok, not WIP anymore.

**Pull request merged**